src/tests/ftest/util/server_utils.py

```diff
@@ -50,6 +57,439 @@ class ServerFailed(Exception):
     """Server didn't start/stop properly."""
 
 
+class DaosServer(DaosCommand):
+    """Defines an object representing a server command."""
+
+    def __init__(self, path=""):
+        """Create a server command object
+
+        Args:
+            path (str): path to location of daos_server binary.
+        """
+        super(DaosServer, self).__init__(
+            "/run/daos_server/*", "daos_server", path)
+
+        self.yaml_params = self.DaosServerConfig()
+        self.timeout = 30
+        self.server_cnt = 1
+        self.mode = "normal"
+
+        self.debug = FormattedParameter("-b", True)
+        self.json = FormattedParameter("-j", False)
+        self.config = FormattedParameter("-o {}")
+
+    def get_params(self, test):
+        """Get params for Server object and server configuration."""
+        super(DaosServer, self).get_params(test)
+        self.yaml_params.get_params(test)
+
+    def set_action_command(self):
+        """Set the action command object based on the yaml provided value."""
```

Comment:
Per the previous command_utils.py comments:
```suggestion
        """Get the action command object based on the yaml provided value."""
```